### PR TITLE
Installation instruction updates

### PR DESCRIPTION
**Pull request by kcpevey**
## What does this implement/fix?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Review by viniciusdc — APPROVED** [review]
LGTM! Thanks @kcpevey

File: docs/docs/getting-started/installing-nebari.md
Comment (trallard) [inline docs/docs/getting-started/installing-nebari.md:10]: This title feels out of place - should this be bumped to the top of the page?

File: docs/docs/getting-started/installing-nebari.md
Comment (trallard) [inline docs/docs/getting-started/installing-nebari.md:53]:
```suggestion
You can verify that the Nebari package is properly installed, and you can execute the client commands by running the following command on your command line:
```

File: docs/docs/how-tos/nebari-aws.md
Comment (trallard) [inline docs/docs/how-tos/nebari-aws.md:110]: Suggestion: split into bullets so that the keys in the config.yml file are easier to parse/read

File: docs/docs/how-tos/nebari-aws.md
Comment (trallard) [inline docs/docs/how-tos/nebari-aws.md:113]: Using `\` and multiline makes it easier to read the command in smaller screens so I'd prefer leaving this as it was before

File: docs/docs/how-tos/nebari-azure.md
Comment (trallard) [inline docs/docs/how-tos/nebari-azure.md:93]: same as above

File: docs/docs/how-tos/nebari-do.md
Comment (trallard) [inline docs/docs/how-tos/nebari-do.md:88]: And here

**Review by trallard — DISMISSED** [review]
Added a few comments - which are applicable pretty much to all the guides

File: docs/docs/how-tos/nebari-aws.md
Comment (kcpevey) [reply docs/docs/how-tos/nebari-aws.md:113]: for people actually following along with this as a guide, they will copy and paste the command. Its very frustrating for this to be multi-line in that case. ( I was frustrated by it when I walked through the guide and Eskild mentioned he found it annoying too)

File: docs/docs/how-tos/nebari-aws.md
Comment (kcpevey) [reply docs/docs/how-tos/nebari-aws.md]: done.

File: docs/docs/how-tos/nebari-do.md
Comment (kcpevey) [reply docs/docs/how-tos/nebari-do.md]: done.

File: docs/docs/how-tos/nebari-azure.md
Comment (kcpevey) [reply docs/docs/how-tos/nebari-azure.md]: done.

File: docs/docs/getting-started/installing-nebari.md
Comment (kcpevey) [reply docs/docs/getting-started/installing-nebari.md]: done locally

File: docs/docs/how-tos/nebari-aws.md
Comment (kcpevey) [reply docs/docs/how-tos/nebari-aws.md]: the `\` makes it easier to read multiline on smaller screens but harder to read on larger screens. So we are targeting the minority. 

This is what it looks like on my laptop 
image

Comment (kcpevey) [reply docs/docs/how-tos/nebari-aws.md]: All that said, I defer to your final decision.

Comment (trallard) [reply docs/docs/how-tos/nebari-aws.md]: > the \ makes it easier to read multiline on smaller screens but harder to read on larger screens. So we are targeting the minority.

Disagree on targeting the minority, smaller screens also count as wide screens being split in thirds/quarters or long format screens. 
Others need to use zoom for low vision concerns which also lead to inaccessible long strings of text, not to mention those who navigate sites with a keyboard only.


Re accessibility - the natural text in (the body of) websites usually has automatic reflow, this is not directly applied to `<pre> <code>`  elements, so splitting the long string here helps as a workaround for the lack of automatic reflow.

I do not think it makes it harder to read on larger screens - it is just a different display format to what some might be used to and many other documentation sites do use this split syntax for long commands, often with many flags. So I prefer keeping this.

Comment (trallard) [reply docs/docs/how-tos/nebari-aws.md]: Also about this in particular

> So we are targeting the minority.
I have very strong feelings about this - it is usually minorities that get underserved / not thought of. So I prefer thinking in terms of the "the-curb-cut-effect" more often than not

Comment (kcpevey) [reply docs/docs/how-tos/nebari-aws.md]: I apologize for using a trigger word. I didn't mean it in that sense. Backing away slowly now....

File: docs/docs/getting-started/installing-nebari.md
Comment (kcpevey) [reply docs/docs/getting-started/installing-nebari.md]: Fixed.

**Comment by kcpevey** [comment]
@trallard merge conflicts have been resolved. Ready for review.

File: docs/docs/how-tos/nebari-azure.md
Comment (pavithraes) [inline docs/docs/how-tos/nebari-azure.md:93]:
```suggestion
2. Executing the command below will generate a basic config file with an infrastructure based on **Azure**. 
```

File: docs/docs/how-tos/nebari-azure.md
Comment (pavithraes) [inline docs/docs/how-tos/nebari-azure.md:102]:
```suggestion
    qhub init azure --project projectname --domain domain --auth-provider password
```

File: docs/docs/how-tos/nebari-do.md
Comment (pavithraes) [inline docs/docs/how-tos/nebari-do.md:88]:
```suggestion
2. Executing the command below will generate a basic config file with an infrastructure based on **DigitalOcean**. 
```

File: docs/docs/how-tos/nebari-do.md
Comment (pavithraes) [inline docs/docs/how-tos/nebari-do.md:97]:
```suggestion
    qhub init do --project projectname --domain domain --auth-provider password
```

File: docs/docs/how-tos/nebari-gcp.md
Comment (pavithraes) [inline docs/docs/how-tos/nebari-gcp.md:131]:
```suggestion
2. Executing the command below will generate a basic config file with an infrastructure based on **GCP**. 
```

File: docs/docs/how-tos/nebari-gcp.md
Comment (pavithraes) [inline docs/docs/how-tos/nebari-gcp.md:140]:
```suggestion
    qhub init gcp --project projectname --domain domain --auth-provider password
```

**Comment by pavithraes** [comment]
@kcpevey I've left some small typo-fix suggestions, other than that, this looks good to me!

**Comment by kcpevey** [comment]
Thanks @pavithraes! Now I think we're just waiting on final review from @trallard

**Comment by trallard** [comment]
I only realised this is waiting for my review - will do this tomorrow

**Comment by trallard** [comment]
I had a quick look, and now it seems these instructions and the ones available in https://www.nebari.dev/docs/get-started/installing-nebari and the various cloud providers conflict with each other - Asmi and @iameskild  updated these docs to match the newest CLI. 

Can rebase `main` and/or sort through the conflicts (see above

**Comment by pavithraes** [comment]
@kcpevey I'm happy to take over this issue and push up commits to resolve conflicts, would you be fine with that?

**Comment by kcpevey** [comment]
@pavithraes yep. she's all yours! :)

**Comment by pavithraes** [comment]
Thank you, `@kcpevey`!

@trallard This PR is ready for your final review. :)

**Pull request merged**
